testing/testcafe/tests/dataGrid/scrolling.ts

```diff
@@ -1006,6 +1006,536 @@ fixture`Remote Scrolling`
     await t.maximizeWindow();
   });
 
+const apiRequestMock = RequestMock()
```

Comment:
Let's move this mock code to a separate file
E.g. to ```testing/testcafe/tests/dataGrid/apiMocks/salesApiMock.ts```

And export this const from this new file:
```typescript
export const salesApiMock = RequestMock()
// other code
```